app/models/worksheet.rb

```diff
@@ -28,4 +28,27 @@ class Worksheet < ApplicationRecord
 
   validates :title, :template_image_url, presence: true
   validates :canvas, length: { minimum: 0, allow_nil: false, message: "can't be nil" }
+
+  def parse_for_dashboard
+    {
+      id: id,
+      title: subject,
+      subject: work_group.classroom.subject,
+      color: get_color,
+      due_date: "#{Date.now + 7}"
+      link: '#'
+    }
+  end
+
+  def get_color
```

Comment:
This get_color method and the other one in the classroom model need to both me removed and combined into a single method, but for now I left them separate just to simplify things and get this PR done faster.

Reply:
Create a DashboardHelper file and move the method there

Reply:
Create a DashboardHelper file and move the method there

Reply:
also 1 more point:
- get_color doesn't sit well with rubocop. Please make sure you have it correctly installed and set up